tests/gold_tests/pluginTest/lua/gold/metrics.gold

```diff
@@ -0,0 +1,6 @@
+plugin.lua.global.states 4
+plugin.lua.global.gc_bytes 249856
```

Comment:
will gc_bytes always be this value? across different platforms?

Reply:
Good point, the original test didn't look for specific numbers.  I'm not at all familiar with the Lua plugin.  I just addressed the problem of getting Au test to wait for stats accumulation and logging output.  Is @traeak still working on ATS?

Reply:
of this whole file the only piece that probably matters is the

plugin.lua.global.states 4

The rest of the numbers don't really matter.  Actually eliminating the gold file and just looking for the above to be there and '4' would be a better way to do it.